src/tpstreams/assets/includes/watch_metrics.html

```
@@ -1,59 +1,151 @@
-<dl class="grid grid-cols-1 mt-6 md:grid-cols-2 xl:grid-cols-1 gap-y-4" x-show="!disableAnalytics">
 
-  <div class="flex flex-none w-full gap-x-4">
-    <dt class="flex-none">
-      <span class="sr-only">Views</span>
-      <svg fill="none" stroke="currentColor" stroke-width="1.5" class="w-6 h-6 text-gray-400" viewBox="0 0 24 24">
-        <path stroke-linecap="round" stroke-linejoin="round" d="M21 12a9 9 0 1 1-18 0 9 9 0 0 1 18 0z" />
-        <path stroke-linecap="round" stroke-linejoin="round"
-          d="M15.91 11.672a.375.375 0 0 1 0 .656l-5.603 3.113a.375.375 0 0 1-.557-.328V8.887c0-.286.307-.466.557-.327l5.603 3.112z" />
-      </svg>
+<div class="m-6 grid grid-cols-3 gap-2 md:gap-2 lg:grid-cols-4 xl:gap-4">
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)